scripts/common/rook.sh

```diff
@@ -399,9 +364,76 @@ EOM
     curl -sSL https://raw.githubusercontent.com/rook/rook/v1.4.9/cluster/examples/kubernetes/ceph/upgrade-from-v1.3-apply.yaml | kubectl apply -f -
     curl -sSL https://raw.githubusercontent.com/rook/rook/v1.4.9/cluster/examples/kubernetes/ceph/upgrade-from-v1.3-crds.yaml | kubectl apply -f -
     kubectl -n rook-ceph set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v1.4.9
-    kubectl -n rook-ceph set image deploy/rook-ceph-tools rook-ceph-tools=rook/ceph:v1.4.9
+    kubectl apply -f << EOM
```

Comment:
now that we have an add-on can we move this and other yaml to separate files?

Reply:
Yep, I'm working on that at the moment. (along with "actually downloading the addon and loading images from it")

Reply:
(I already have the yaml committed, I just don't have the way to access it yet)